scripts/scripts.js

```diff
@@ -5,6 +5,8 @@ function Clock() {
     let timeZoneSign = timeZoneOffsetInMinutes > 0 ? '-' : '+';
     let timeZoneOffsetHours = Math.abs(Math.floor(timeZoneOffsetInMinutes / 60));
 
+    let utcStyledString = `${time.toISOString().split(".")[0]}.000Z`;
```

Comment:
First of all, it wasn't stupidity 😄. And secondly, I think it would be better if we do so `new Date(unix * 1000).toISOString();`. I'll make a commit now